app/views/data_exports/_manifest.html.erb

```diff
@@ -0,0 +1,68 @@
+<ul class="space-y-4 list-inside list-none mt-2">
+  <% @manifest['children'].each do |project| %>
+   <li>
+      <div class="flex text-primary-800 dark:text-primary-400">
+        <%= viral_icon(name: "folder", classes: "w-5 h-5 m-0") %>
```

Comment:
**nitpick:** I find the font sizing on this page a little large, can you play with it a bit and see how it looks?

Reply:
Used text-sm e6e2a039de410dc1535686ca33df7ac42470e1eb